src/routes/matchHistoryRoutes.js

```diff
@@ -0,0 +1,18 @@
+import express from 'express'
+
+function getSummoner(){
+    const router = express.Router()
+    router.get('/summoner', getSummonerHandler)
+    router.get('/summoner/:id', getSummonerNameHandler)
```

Comment:
Let's use the getSummonerHandler like this:
```suggestion
    router.get('/summoner/:summonerName', getSummonerHandler)
```